inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/redis/RedisSinkOperator.java

```diff
@@ -0,0 +1,159 @@
+import org.apache.inlong.manager.pojo.sink.SinkRequest;
+import org.apache.inlong.manager.pojo.sink.StreamSink;
+import org.apache.inlong.manager.pojo.sink.redis.RedisClusterMode;
+import org.apache.inlong.manager.pojo.sink.redis.RedisDataType;
+import org.apache.inlong.manager.pojo.sink.redis.RedisSchemaMapMode;
+import org.apache.inlong.manager.pojo.sink.redis.RedisSink;
+import org.apache.inlong.manager.pojo.sink.redis.RedisSinkDTO;
+import org.apache.inlong.manager.pojo.sink.redis.RedisSinkRequest;
+import org.apache.inlong.manager.service.sink.AbstractSinkOperator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Redis sink operator, such as save or update redis field, etc.
+ */
+@Service
+public class RedisSinkOperator extends AbstractSinkOperator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RedisSinkOperator.class);
+    private static final int PORT_MAX_VALUE = 65535;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Override
+    public Boolean accept(String sinkType) {
+        return SinkType.REDIS.equals(sinkType);
+    }
+
+    @Override
+    protected String getSinkType() {
+        return SinkType.REDIS;
+    }
+
+    @Override
+    protected void setTargetEntity(SinkRequest request, StreamSinkEntity targetEntity) {
+
+        if (!this.getSinkType().equals(request.getSinkType())) {
+            throw new BusinessException(ErrorCodeEnum.SINK_SAVE_FAILED,
+                    SINK_TYPE_NOT_SUPPORT.getMessage() + ": " + getSinkType());
+        }
+
+        RedisSinkRequest sinkRequest = (RedisSinkRequest) request;
+
+        String clusterMode = sinkRequest.getClusterMode();
+        RedisClusterMode redisClusterMode = RedisClusterMode.of(clusterMode);
+
+        expectNotNull(redisClusterMode,
+                "Redis ClusterMode must in one of " + Arrays.toString(RedisClusterMode.values()) + " !");
+
+        switch (redisClusterMode) {
+            case CLUSTER:
+                String clusterNodes = sinkRequest.getClusterNodes();
+                checkClusterNodes(clusterNodes);
+                break;
+            case SENTINEL:
+                String sentinelMasterName = sinkRequest.getSentinelMasterName();
+                expectNotEmpty(sentinelMasterName, "Redis MasterName of Sentinel cluster must not null!");
+                String sentinelsInfo = sinkRequest.getSentinelsInfo();
+                expectNotEmpty(sentinelsInfo, "Redis sentinelsInfo of Sentinel cluster must not null!");
+                break;
+            case STANDALONE:
+                String host = sinkRequest.getHost();
+                Integer port = sinkRequest.getPort();
+
+                expectNotEmpty(host, "Redis server host must not null!");
+                expectTrue(
```

Comment:
`expectTrue(port != null && port > 1 && port < PORT_MAX_VALUE,
                        "The port of the redis server must be greater than 0 and less than 65535!")`